aspnetcore/fundamentals/hosted-services/sample/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace BackgroundTasksSample
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            BuildWebHost(args).Run();
+        }
+
+        public static IWebHost BuildWebHost(string[] args) =>
+            WebHost.CreateDefaultBuilder(args)
+                .UseStartup<Startup>()
+                .ConfigureLogging((hostingContext, logging) =>
+                {
+                    // Remove the Debug provider (and other providers) and filter most
```

Comment:
Why are we doing this in a sample? It seems completely unrelated to the content.

Reply:
To clean up the console messages so that the messages written by the methods we're showing don't get lost. We do this on the logger message message topic, too, for the same effect.

Reply:
Ugh

Reply:
The alternative is to pull the lines and let the dev search for the messages. Would u prefer to do that? I'll strike this here and over in the Logger Message topic.